adminSiteServer/apiRouter.ts

```diff
@@ -2276,13 +2276,19 @@ deleteRouteWithRWTransaction(apiRouter, "/gdocs/:id", async (req, res, trx) => {
     // Assets have TTL of one week in Cloudflare. Add a redirect to make sure
     // the page is no longer accessible.
     // https://developers.cloudflare.com/pages/configuration/serving-pages/#asset-retention
-    await db.knexRawInsert(
-        trx,
-        `INSERT INTO redirects (source, target, ttl)
-         VALUES (?, ?, DATE_ADD(NOW(), INTERVAL 8 DAY))`,
-        [getCanonicalUrl("", gdoc), "/"]
-    )
-    await triggerStaticBuild(res.locals.user, `Deleting ${gdoc.slug}`)
+    const gdocSlug = getCanonicalUrl("", gdoc)
```

Comment:
So, if I see this correctly, if `gdoc` is a fragment then we'll end up with `gdocSlug = ""`. We should explicitly handle that case, too - don't know what currently happens if we ever insert an empty string into the table.

Reply:
Looking at it, it'll probably generate an incorrect redirect rule (something like ` / 302`) which CF will then ignore. Still not great.

Reply:
So rigorous! 😍 

Went with a falsey check instead of an exception for `type == Fragment ` 👍